sortedcontainers/sortedlist.pyi

```diff
@@ -0,0 +1,170 @@
+import typing
+from typing import (
+    Any,
+    Callable,
+    Generic,
+    Iterable,
+    Iterator,
+    List,
+    MutableSequence,
+    Optional,
+    Tuple,
+    Type,
+    TypeVar,
+    Union,
+)
+
+_T = TypeVar("_T")
+_SL = TypeVar("_SL", bound="SortedList")
```

Comment:
`SortedList` and `SortedKeyList` don't need quotes around them. In `pyi` files, forward references work without quotes.